Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -206,6 +207,10 @@ extension XcodeModuleProtocol  {
             }
         }
 
+        if let pkgArgs = try? self.pkgConfigArgs() { 
+            buildSettings["OTHER_CFLAGS"] = serializeArray(pkgArgs.xcc)
```

Comment:
Note that `OTHER_CFLAGS` aren't actually passed to the Swift compiler by Xcode... this probably only works for linking arguments.